CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.10.0
+
+### Added
+*  Recommendation in `getting-started.md` that OneFuzz users include a `.onefuzz` file in their project root directory for security tool detection [#2236](https://github.com/microsoft/onefuzz/pull/2236)
+* Agent:  New `libfuzzer_dotnet_fuzz` task [#2221](https://github.com/microsoft/onefuzz/pull/2221)
+* CLI: Allow dotnet endpoint and functions configuration in the CLI [#2190](https://github.com/microsoft/onefuzz/pull/2190)
+
+### Changed
+* Agent/CLI: Updated default Windows VM image [#2226](https://github.com/microsoft/onefuzz/pull/2226)
+* Agent: Modified LibFuzzer struct to own its environment and option data [#2219](https://github.com/microsoft/onefuzz/pull/2219)
+* Agent: Factor out generic LibFuzzer task [#2214](https://github.com/microsoft/onefuzz/pull/2214)
+* Deployment: Keep both C# and python http functions active when deploying, this will allow the service to be backward compatible with older versions of the agent and the CLI when enabling new C# funcitons on a deployment [#2212](https://github.com/microsoft/onefuzz/pull/2212)
```

Comment:
```suggestion
```
Suggest that we don't put this in because this is just a transition situation while we port to C#.